Guardfile

```diff
@@ -0,0 +1,73 @@
+# A sample Guardfile
```

Comment:
So this is a watcher for frame_embed_generator.rb? Please document the function of this file.

Reply:
I have removed Guard, and watching using foreman for spec files. As I still have not learnt how to write tests especially for Jekyll Plugins.

On the contrary to writing rspec for a simple functionality like so - https://github.com/auto-wcag/auto-wcag-deployer/blob/master/spec/server_spec.rb